README.md

```diff
@@ -1,10 +1,15 @@
-## Kevin_Payment
+# kevin. Magento 2 module
 
 Payments collection solution to your Magento 2 e-commerce platform.
 
 BEFORE YOU START MAKE SURE YOU HAVE RECEIVED CLIENT ID AND CLIENT SECRET FROM KEVIN.
 
-### Installation
+## Prerequisites
+
+- Magento 2.x
+- PHP 7.0 or later

```

Comment:
Is it true? Or just our module is not compactable? 

https://github.com/getkevin/kevin-magento2/pull/3/files/68a09e5e107c390ce4399a1a04162ec1a457d226#r786970178

Reply:
The module seems to be using syntax that was introduced in PHP 7 occasionally so for now it's a good idea to keep the requirement this way. Also, Magento 2 does not really support PHP 5 anymore https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements.html